internal/operator-controller/applier/helm_test.go

```diff
@@ -106,6 +110,10 @@ metadata:
     olm.properties: '[{"type":"from-csv-annotations-key", "value":"from-csv-annotations-value"}]'
 spec:
   installModes:
+    - type: SingleNamespace
+      supported: true
+    - type: OwnNamespace
+      supported: true
```

Comment:
Should not those still fail when the flag is not used and only work when the feature flag is used?

Reply:
Only if you try to install a bundle in Single- OwnNamespace mode. It's ok if a bundle supports those install modes. As it is now for v1, AllNamespaces MUST be supported (but it doesn't need to be the _only_ supported install mode).

I've added another unit test for the GetWatchNamespace function behavior when the FG is off.